packages/front-end/components/License/ShowLicenseInfo.tsx

```diff
@@ -0,0 +1,143 @@
+          <div className="col-sm-3">
+            <h4>License</h4>
+          </div>
+          <div className="col-sm-9">
+            <div className="form-group row mb-2">
+              <div className="col-sm-12">
+                <strong>Plan type: </strong> {licensePlanText}{" "}
+              </div>
+            </div>
+            {showUpgradeButton && (
+              <div className="form-group row mb-1">
+                <div className="col-sm-12">
+                  <button
+                    className="btn btn-premium font-weight-normal"
+                    onClick={() => setUpgradeModal(true)}
+                  >
+                    {accountPlan === "oss" ? (
+                      <>
+                        Try Enterprise <GBPremiumBadge />
+                      </>
+                    ) : (
+                      <>
+                        Try Pro <GBPremiumBadge />
+                      </>
+                    )}
+                  </button>
+                </div>
+              </div>
+            )}
+            {!isCloud() && permissions.manageBilling && (
+              <div className="form-group row mt-3 mb-0">
+                {showInput && (
+                  <div className="col-sm-4">
+                    <div>
+                      <strong>License Key: </strong>
+                    </div>
+                    <div
+                      className="d-inline-block mt-1 mb-2 text-center text-muted"
+                      style={{
+                        width: 100,
+                        borderBottom: "1px solid #cccccc",
+                        pointerEvents: "none",
+                        overflow: "hidden",
+                        verticalAlign: "top",
+                      }}
+                    >
+                      {license ? "***************" : "(none)"}
+                    </div>{" "}
+                    <a
+                      href="#"
+                      className="pl-1"
+                      onClick={(e) => {
+                        e.preventDefault();
+                        setEditLicenseOpen(true);
+                      }}
+                    >
+                      <FaPencilAlt />
+                    </a>
+                  </div>
+                )}
+                {license && (
+                  <>
+                    <div className="col-sm-2">
+                      <div>Issued:</div>
+                      <span className="text-muted">
+                        {date(license.dateCreated)}
+                      </span>
+                    </div>
+                    <div className="col-sm-2">
+                      <div>Expires:</div>
+                      <span className="text-muted">
+                        {date(license.dateExpires)}
+                      </span>
+                    </div>
+                    <div className="col-sm-2">
+                      <div>Seats:</div>
+                      <span className="text-muted">{license.seats}</span>
```

Comment:
Small nit: Given that we default all trial license keys generated via the app to 999 seats. I wonder if we want to add a tooltip here that adds some context about where 999 came from and how when the trial ends they won't necessarily be billed for 999 seats.

I suppose we'd only want to show the tooltip for orgs in a trial with 999 seats, since some orgs will start a trial via sales that will generate the license key for them and define a pre-determined amount of seats.